controllers/kbsconfig_controller.go

```diff
@@ -18,6 +18,7 @@ package controllers
 
 import (
 	"context"
+	std_errors "errors"
```

Comment:
nit: I don't think there is any need to refer `errors` mod as `std_errors`

Reply:
`errors` is clashing with the existing import `k8s.io/apimachinery/pkg/api/errors`. I wanted to avoid using the k8s specific errors, aren't they supposed to be returned by the server API?

Reply:
You can import k8s errors as 
`k8serrors "k8s.io/apimachinery/pkg/api/errors"`

Reply:
While both are ok, the above is the convention you can see..